portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,69 @@
 <html>
```

Comment:
Consider adding a lang="en" in this tag: https://www.w3.org/International/questions/qa-html-language-declarations

This is important for assistive technology to know the language of the content.